Editor/SettingsAnalysis/PlayerSettingsAnalyzer.cs

```diff
@@ -36,8 +36,13 @@ class PlayerSettingsAnalyzer : ISettingsModuleAnalyzer
             "Audio: Speaker Mode",
             new[] { Area.BuildSize },
             "<b>UnityEngine.AudioSettings.speakerMode</b> is not set to <b>UnityEngine.AudioSpeakerMode.Mono</b>. The generated build will be larger than necessary.",
-            "Change <b>Project Settings ➔ Audio ➔ Default Speaker Mode</b> to <b>Mono</b> to reduce the size of the build on disk.");
-
+            "To reduce runtime memory consumption of AudioClips change <b>Project Settings ➔ Audio ➔ Default Speaker Mode</b> to <b>Mono</b>. This will reduce half memory usage of stereo AudioClips. It is also recommended considering enabling the <b>Force To Mono</b> AudioClip import setting to reduce import times and build size.")
```

Comment:
Only a grammar point, not code:

The second sentence looks like it doesn't need to say "reduce" since "half" is a verb:

"This will half memory usage of stereo AudioClips."

Or alternative, just a bit longer:
"This will reduce memory usage of stereo AudioClips by half."

Reply:
Thanks. Original sentence was "this will half memory". I didn't know half is a verb, so I added the reduce word. I'm removing it.